### PR TITLE
Add builder for GTK3

### PR DESCRIPTION
**Pull request by giordano**
<s>This is only a stub of the GTK3 builder and it's only for macOS.</s>  Ref #62

### CODE REVIEW

**Comment by giordano** [comment]
With a bit of pain, I managed to build GTK for macOS.  Some remarks:

* it requires SDK 10.11, currently BinaryBuilder provides v10.10
* I used `./configure && make && make install` instead of meson
* I had to remove the `demos` directory from `SRC_SUBDIRS`

File: G/GTK3/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/gtk+-*/
+
+# We need to run some commands with a native Glib
+apk add glib-dev gtk+3.0
+# This is awful, I know
+ln -sf /usr/bin/glib-compile-resources ${prefix}/bin/glib-compile-resources
+ln -sf /usr/bin/glib-compile-schemas ${prefix}/bin/glib-compile-schemas
+ln -sf /usr/bin/gdk-pixbuf-pixdata ${prefix}/bin/gdk-pixbuf-pixdata
```
Comment (staticfloat) [inline G/GTK3/build_tarballs.jl]: Just a quick note that you'll need to remove these after `make install` so that they don't accidentally get picked up.

**Comment by giordano** [comment]
I think this should be ready to go, after #150 and [registering all pending jll packages](https://github.com/JuliaRegistries/General/pulls?q=is%3Apr+is%3Aopen+author%3Astaticfloat+sort%3Aupdated-desc) :slightly_smiling_face:  (I couldn't test because of all the missing dependencies)

**Comment by staticfloat** [comment]
*gulp* here we go.....

**Pull request merged**